nil/services/nilservice/service.go

```diff
@@ -195,12 +195,16 @@ func getRawApi(
 	database db.DB,
```

Comment:
Commit message of this patch (Changing the way we specify that we should use the readonly API) should contain some description/motivation regarding this change